framework/wazuh/core/tests/data/configuration/default/journald.conf

```diff
@@ -0,0 +1,6 @@
+<localfile>
+   <location>journald</location>
+   <log_format>journald</log_format>
+   <filter field="MESSAGE_ID">^8d45620c1a4348dbb17410da57c60c66$</filter>
+   <only-future-events>no</only-future-events>
+</localfile>
```

Comment:
```suggestion
</localfile>

```

Reply:
Added in ba463c2b3f963a736c4a8f466f6694363fe12bc6.